### PR TITLE
List command changes: fix typo in "stats", clarify error message

### PR DESCRIPTION
**Pull request by Jurph**
Added handler for "!list options" and fixed typo on 'stats' in IF/ELIF block

### CODE REVIEW

File: rpg/game.py [hunk 1]
```diff
@@ -138,16 +138,18 @@ def pub_command(self, player, command):
...
                 elif things == 'conditions':
                     return CharCond.list()
+                elif things == 'options':
+                    return ListOptions.list()    
                 else:
-                    return '[[ I do not know any of those. ]]'
+                    return '[[ I do not know any of those; try "!list options". ]]'
             except (KeyError, IndexError):
-                return '[[ List needs another parameter. ]]'
+                return '[[ List needs another parameter; try "!list options". ]]'
```
Comment (czeano) [inline rpg/game.py]: It wasn't evident at the time, but I'm trying to put all message/error strings in a localization package. Those are placeholders that will be abstracted out later. (And I think I caught them in the pass that's in #18 )  The place to edit messages is /util/gamedata_dump.py -- until that gets cleaned up and becomes more modular

**Comment by czeano** [comment]
Your change suggesting "!list options" does not correspond to functionality that was added in this branch.  That commit should be part of the other branch (and other pull request).  Otherwise, there's a chance this pull request could be merged adding the error message but not the one that adds the ListOptions.... which would just be mean to users.

**Comment by Jurph** [comment]
I'm still figuring out how to (e.g.) make multiple changed files part of a single pull request. 

I get that `Pull req : git :: config change request : config mgmt board` but a lot of git's functionality on the web interface is... not intuitive?  I'm also new.  I'll keep poking around.  You can expect to see a lot of bumbling.

**Comment by czeano** [comment]
Ahhhhh that explains the commit messages of "Update filename" which aren't exactly descriptive. Yeah, the web interface isn't great for code changes. It *is* the best way to submit a pull request, though.  Relatedly: Thanks for doing these changes on a branch instead of directly into master. 

One commit should contain a cohesive set of changes (as few as 1 change to 1 file) that accomplish "something" atomically.  In this case, say, a commit for updating the !list command and a commit (or several) for implementing the ListOptions class.

A branch should contain a set of commits that are related and disruptively change the application. In this case, all your commits related to implementing !list options.

A pull request should be for one "completed" branch into the working version of the application.  Completed usually meaning "it implements everything I think it should" not necessarily "fully tested and working."  The PR phase can include bug fixes or functionality tweaks based on peer review. 

See http://nvie.com/posts/a-successful-git-branching-model/ for a great template for git behavior.

**Comment by Jurph** [comment]
> One commit should contain a cohesive set of changes (as few as 1 change
to 1 file) that accomplish "something" atomically.
> A branch should contain a set of commits that are related and
disruptively change the application.

Yeah; conceptually this is identical to minor/major/critical changes in any
Config Mgmt Review Board for a major engineering effort -- one Thing
changes, and all the downstream things that are implied also change, and
all of that goes in one System Change Request ("ticket", "CR",
"other_tla"...) .  Same with tickets in our software system, but I'm
usually on the 'putting tickets in' side of the house.

I broke a canonical rule fixing the "statss" typo during a feature fix.
(One CRB I was on actually had a "other cleanup items" box where you could
list, by reference, other fixes with an impact of 'cosmetic' that you
performed or proposed.)

I'll read up on the branching model before I get any more paint on the
furniture or the dog.

cheers
JR

**Pull request closed without merging**